src/components/ProjectForm/ProjectForm.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import CreatableSelect from 'react-select/creatable';
+import { Form, Label, Input, TextArea } from './styles';
+import getAllAndis from '../../api/handlers/andis';
+import getAllTechnologies from '../../api/handlers/technologies';
+import getAllClients from '../../api/handlers/clients';
+
+const ProjectForm = () => {
+
+    const fetchAndis = async () => {
+        const response = await getAllAndis();
+
+        response.data.res.forEach(element => {
+          let andiValues = { value: element["andiid"], label: element["firstname"] };
+          andiOptions.push(andiValues);
+        });
+    }
+
+    const fetchTechnologies = async () => {
+        const response = await getAllTechnologies();
+
+        response.data.res.forEach(element => {
+          let technologyValues = { value: element["technologyid"], label: element["name"] };
+          technologyOptions.push(technologyValues);
+        });
+    }
+
+    const fetchClients = async () => {
+        const response = await getAllClients();
+        response.data.res.forEach(element => {
+          let clientsValues = { value: element["clientid"], label: element["clientname"] };
+          clientOptions.push(clientsValues);
+        });
+    }
```

Comment:
We should look at adding error handling here - My thinking is that if any request fails, we display an error and not the form - don't want users adding projects without the correct andis, tech-stacks etc

Reply:
Agreed, this will crash if response is null or doesn't have a data element